### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by garyb** [natefaubion/purescript-tidy#13]
``` 
C:\Users\Gary\AppData\Roaming\npm\node_modules\purs-tidy\output\Effect.Aff\foreign.js:532
                throw util.fromLeft(step);
                ^
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'C:UsersGaryAppDataRoaming
pm
ode_modulespurs-tidyoutputMainindex.js'
Require stack:
- W:\Projects\Test\[worker eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at [worker eval]:1:1
    at Script.runInThisContext (vm.js:133:18)
    at Object.runInThisContext (vm.js:310:38)
    at internal/process/execution.js:77:19
    at [worker eval]-wrapper:6:22
    at evalScript (internal/process/execution.js:76:60) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'W:\\Projects\\Test\\[worker eval]' ]
}
```
😆 

(I'll take a look at fixing this)

**Comment by natefaubion** [comment]
Oh no! I wonder if this is an issue with node-workerbees.

**Comment by natefaubion** [comment]
https://github.com/natefaubion/purescript-node-workerbees/blob/d8eb29e0eee087c3d136b3b961074c8c11fbbfdd/src/Node/WorkerBees.js#L12
https://github.com/natefaubion/purescript-node-workerbees/blob/d8eb29e0eee087c3d136b3b961074c8c11fbbfdd/src/Node/WorkerBees.js#L84
https://github.com/natefaubion/purescript-node-workerbees/blob/d8eb29e0eee087c3d136b3b961074c8c11fbbfdd/src/Node/WorkerBees.js#L47-L51

**Comment by garyb** [comment]
Ah yeah, I think those are likely culprits. Given what it's printing it's the windows path delimiters being treated as string escapes. I'll try some things out, but the easiest fix might just be to substitute `\` for `/` when printing those values, as I think windows is happy with that. Will try things out/submit a PR later on!

**Comment by natefaubion** [comment]
Ah! Yeah that sounds likely! Does node have a built-in way to escape paths?

**Comment by garyb** [comment]
Nothing that would help here I think, although checking `path.sep` could be used to determine whether it needs double-escaping or something - I think it's just because here you're producing a string of javascript for the worker in that first case you linked to, so the worker is seeing `require("whatever\node_modules\...")` rather than `require("whatever\\node_modules\\...")`

**Comment by wclr** [comment]
Yeah, `res.filePath.replace(/\\/g, '\\\\' )` in WorkerBees.js fixes it.

**Comment by natefaubion** [comment]
This was fixed in the 0.1.1 release.
